rpm/fedora-37/Dockerfile

```diff
@@ -0,0 +1,33 @@
+ARG GO_IMAGE
+ARG DISTRO=fedora
+ARG SUITE=37
+ARG BUILD_IMAGE=${DISTRO}:${SUITE}
+
+FROM ${GO_IMAGE} AS golang
+
+FROM ${BUILD_IMAGE}
+ENV GOPROXY=direct
+ENV GO111MODULE=off
+ENV GOPATH /go
+ENV PATH $PATH:/usr/local/go/bin:$GOPATH/bin
+ENV AUTO_GOPATH 1
+ENV DOCKER_BUILDTAGS seccomp selinux
+ENV RUNC_BUILDTAGS seccomp selinux
```

Comment:
Added these back in this branch; the `RUNC_BUILDTAGS` is probably not needed, but the `DOCKER_BUILDTAGS` is still used for 20.10